azext_iot/operations/events3/_events.py

```diff
@@ -142,7 +150,18 @@ def _output_msg_kpi(msg):
             if app_prop:
                 event_source['event']['properties']['application'] = unicode_binary_map(app_prop)
 
-        six.print_(yaml.dump(event_source, default_flow_style=False), flush=True)
+        if output and output == 'json':
+            try:
+                if event_source['event']['payload']:
+                    event_source['event']['payload'] = json.loads(event_source['event']['payload'].replace('\\n', '').replace('\\r', ''))
```

Comment:
Do you think we should RegEx this?

Reply:
Yes I do...a compiled pattern would fit the bill...something like:

```python
pattern = re.compile(r'\\r\\n|\\r|\\n')
...
...
...
if event_source['event']['payload']:
    search_result = pattern.search(event_source['event']['payload'])
    if search_result:
        event_source['event']['payload'] = event_source['event']['payload'].replace('\\n', '').replace('\\r', '')
```